jekyll-sass-converter.gemspec

```
@@ -14,9 +14,9 @@ Gem::Specification.new do |spec|
   spec.files         = `git ls-files -z`.split("\x0").grep(%r!^lib/!)
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = ">= 2.6.0"
+  spec.required_ruby_version = ">= 3.1"
 
-  spec.add_runtime_dependency "sass-embedded", "~> 1.54"
+  spec.add_runtime_dependency "sass-embedded", "~> 1.80"
```

If we want to allow 1.75 and above only, we need to tack on an extra constraint to the dependency. Writing this from my phone, so not sure if this is the actual syntax.

```suggestion
  spec.add_runtime_dependency "sass-embedded", "~> 1.80, >=1.75"
```

---

Technically, this suggestion would be redundant / incorrect since `~> 1.80` equates to `>= 1.80` and therefore doesn't include `<= 1.79.x`. 
The simplest declaration would be:
```diff
-  spec.add_runtime_dependency "sass-embedded", "~> 1.80"
+  spec.add_runtime_dependency "sass-embedded", "~> 1.75"
```

The *tack-on extra constraint* would be applicable to filter already included versions:
```diff
-  spec.add_runtime_dependency "sass-embedded", "~> 1.80"
+  spec.add_runtime_dependency "sass-embedded", "~> 1.74", ">= 1.74.1"   # block v1.74.0
```